cloudflare/src/endpoints/plan.rs

```diff
@@ -5,6 +5,9 @@ pub enum Frequency {
     Monthly,
     Quarterly,
     Yearly,
+    /// We get an empty variant for free plans
+    #[serde(other)]
+    Missing,
```

Comment:
Rather than add a new case for missing, I'd rather structs just use Option<Frequency> instead, and update the docstring for this enum to say "Free plans won't have a Frequency, so most responses should accept Option<Frequency> instead."

Reply:
Done